docs/OMF.rst

```diff
@@ -179,28 +77,174 @@ Connect the new application to the PI Connector Relay by selecting the new Fledg
+   
+   - **PI Web API Kerberos keytab file**: The Kerberos keytab file used to authenticate.
+
+Cloud
+~~~~~
+
+The *Cloud* tab contains configuration items that are required if the chosen OMF end point is either AVEVA Data Hub or OSISoft Cloud Services.
+
++-------------+
+| |OMF_Cloud| |
++-------------+
+
+  - **Cloud Service Region**: - The region in which your AVEVA Data Hub or OSISoft Cloud Services service is located.
+
+    +---------------+
+    | |ADH_Regions| |
+    +---------------+
+
+  - **Namespace**: Your namespace within the AVEVA Data Hub or OSISoft Cloud Service.
+
+  - **Tenant ID**: Your AVEVA Data Hub or OSISoft Cloud Services Tenant ID for your account.
+
+  - **Client ID**: Your AVEVA Data Hub or OSISoft Cloud Services Client ID for your account.
+
+  - **Client Secret**: Your AVEVA Data Hub or OSISoft Cloud Services Client Secret.
+
+Connection
+~~~~~~~~~~
+
+The *Connection* tab allows a set of tuning parameters to be set for the connection from the OMF plugin to the OMF End point.
+
++------------------+
+| |OMF_Connection| |
++------------------+
+
+
+   - **Sleep Time Retry**: Number of seconds to wait before retrying the connection (Fledge doubles this time after each failed attempt).
+
+   - **Maximum Retry**: Maximum number of times to retry connecting to the OMF Endpoint.
+
+   - **HTTP Timeout**: Number of seconds to wait before Fledge will time out an HTTP connection attempt.
+
+   - **Compression**: Compress the readings data before sending them to the OMF endpoint.
+
+Formats & Types
+~~~~~~~~~~~~~~~
+
+The *Formats & Types* tab provides a means to specify the detail types that will be used and the way complex assets are mapped to OMF types to also be configured.
+
++--------------+
+| |OMF_Format| |
++--------------+
+
+   - **Integer Format**: Used to match Fledge data types to the data type configured in PI. This defaults to int64 but may be set to any OMF data type compatible with integer data, e.g. int32.
+
+   - **Number Format**: Used to match Fledge data types to the data type configured in PI. The default is float64 but may be set to any OMF datatype that supports floating point values.
+
+   - **Complex Types**: Versions of the OMF plugin prior to 2.1 support complex types in which each asset would have a corresponding OMF type created for it. With the introduction of OMF Version 1.2 support in version 2.1.0 of the plugin support has been added for linked types. These are more versatile and allow for asset structures to change dynamically. The linked types are now the default, however setting this option can force the older complex types to be used.  See :ref:`Linked_Types`. Versions of the PI Server from 2020 or before will always use the complex types. The plugin will normally automatically detect this, however if the detection does not correctly enforce this setting then this opion should be enabled by the user.
```

Comment:
Typo: this _opion_ should be enabled by the user -> this _option_ should be enabled by the user